### ISSUE TITLE
issue: Form validation doesn't work when field is registered on disabled form

### ISSUE THREAD

**Issue by AlexKrupko** [react-hook-form/react-hook-form#12285]
### Version Number

7.53.0

### Codesandbox/Expo snack

https://codesandbox.io/p/sandbox/headless-cdn-d55rpz

### Steps to reproduce

1. Go to linked Codesandbox example
2. Uncheck the "Disabled form" checkbox
3. Fill out the input field
4. Submit the form 

### Expected behaviour

Form submit method should be fired, field should be valid.

**Version 7.52.2 doesn't have this issue.** 

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
should set disabled prop on the useController.

**Comment by AlexKrupko** [comment]
Yes, I can take the `disabled` prop from `useFormState` and pass it to `useController`. I've tested it and it works. 
But could you describe this in the documentation? There is no mentions at all about the `disabled` prop for `useForm` hook.
